### PR TITLE
Project refactoring suggestions

### PR DESCRIPTION
**Pull request by maryzam**
Hello!

I really like this project and would be glad to contribute to it's source code and help you to fix bugs and add new features.

That why I've made some changes in the project structure to make it more readable, easy-to-test and easy-to-merge for other developers. For now I've just separate different classes from TwitterJson.cs to separate files to verify that you're interested in such contributions.

However, I've got some more ideas how to make the project better (such as adding unit tests, change some class names to avoid misunderstanding (DeleteStatus and StatusDelete for example), etc.

Thanks!

### CODE REVIEW

**Comment by toofusan** [comment]
Sorry for late. Thanks for your suggestion !
I'll check it soon. Please wait.

File: DataModels/DirectMessages/DirectMessages.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (toofusan) [inline DataModels/DirectMessages/DirectMessages.cs]: In my envorinment, There are some errors about `System.Threading.Tasks`.
```
/DataModels/DirectMessages/DirectMessages.cs(5,24): error CS0234: The type or namespace name `Tasks' does not exist in the namespace `System.Threading'. Are you missing an assembly reference?
```
Now I use Unity 2017.2.0.f3. Do you have any idea about this error?

**Comment by maryzam** [comment]
Hello, @toofusan! 
Thank you for your response.

You're right, there was an issue with some "default" namespaces that Visual Studio adds to every new file. Sorry about it!

I've removed all useless namespaces and now it should work properly.
P.S. I'm also using the same version of Unity now.

**Comment by toofusan** [comment]
I have confirmed that new branch works properly.
Thanks for your contribution!

**Pull request merged**